modules/base/rendering/renderableplane.h

```diff
@@ -73,6 +73,10 @@ class RenderablePlane : public Renderable {
     properties::BoolProperty _mirrorBackside;
     properties::Vec2Property _size;
     properties::BoolProperty _autoScale;
+    properties::BoolProperty _scaleByDistance;
+    properties::FloatProperty _scaleRatio;
+    properties::FloatProperty _scaleByDistanceMaxHeight;
+    properties::FloatProperty _scaleByDistanceMinHeight;
```

Comment:
Since all these properties are related, how about collecting them in a subpropertyowner?